src/storage/executor/outcome.rs

```diff
@@ -61,6 +61,8 @@ pub enum OutcomePrefix {
     CreateTransactionSuccess = 0xd0,
     TransactionCommitSuccess = 0xd1,
     TransactionAbortSuccess = 0xd2,
+
+    ServerError = 0xf0,
```

Comment:
`Outcome` was designed to convey success, and error would become `ImmuxError`.
We either continue this tradition, or we merge all possible errors into Outcome. I don't the adding one `ServerError` on `Outcome` is a consistent design.

Reply:
ref to the major refactoring.